apps/cpp_rpc/rpc_env.cc

```diff
@@ -95,7 +96,16 @@ RPCEnv::RPCEnv(const std::string& wd) {
     auto cmdline = fopen("/proc/self/cmdline", "r");
     fread(cwd, 1, sizeof(cwd), cmdline);
     fclose(cmdline);
-    base_ = "/data/data/" + std::string(cwd) + "/cache/rpc";
+    std::string android_base_ = "/data/data/" + std::string(cwd) + "/cache";
+    struct stat statbuf;
+    // Check if application data directory exist. If not exist usually mean tvm_rpc run from adb
```

Comment:
Nitty comment. `If not exist, usually means we run tvm_rpc from adb shell terminal`